yt/yt/library/auth_server/config.h

```diff
@@ -232,6 +250,12 @@ class TOAuthServiceConfig
     std::optional<TString> UserInfoSubjectField;
     std::optional<TString> UserInfoErrorField;
 
+    //! If specified, configures a global regex replace transformation on the
+    //! contents of the login field. It will replace all non-overlapping matching
+    //! strings with the replacement string. You can use \1-\9 for captured match
+    //! groups in the replacement string, and \0 for the whole match.
+    TStringReplacementConfigPtr LoginTransformation;
```

Comment:
Maybe we should give user a hint what type of regex standard (PCRE, POSIX, etc) will be used to interpret his regex in our lib, so he wouldn't have to guess?

Reply:
It's RE2 [syntax](https://github.com/google/re2/wiki/Syntax). Judging from this page, it is "a subset of that accepted by PCRE, roughly speaking, and with various [caveats](https://swtch.com/~rsc/regexp/regexp3.html#caveats)."

I will mention it in the comment.